src/sections/NavBar/NavBar.scss

```diff
@@ -28,6 +27,7 @@
   }
   
   &-links-container {
+    padding: 0 100px;
```

Comment:
This fixed padding causes problems at smaller screen sizes. Starting at a certain point, it starts getting unaligned and becomes more and more noticeable.